### DISCUSSION TITLE
[Bug Report] Slots 插槽获得当前本条数据或者当前的本条数据的index值

### DISCUSSION THREAD

**Discussion by zhezhe-Arthur** [youzan/vant#11953] [Q&A] [answered]
### 重现链接

https://gitee.com/zhepeng/device-web

### Vant 版本

2.12.54

### 描述一下你遇到的问题。

AddressList 地址列表 使用了Slots 插槽tag后发现不知道如何拿到当前本条数据或者当前的本条数据的index值

### 重现步骤

maintainance页面数据列表 device-web\src\view\maintainance\index.vue

### 设备/浏览器

app

**Comment by inottn** [comment, marked as answer]
可以通过作用域插槽获取
